### PR TITLE
Add -amin, -cmin and -mmin switches

### PR DESCRIPTION
**Pull request by timgimyee**
Add -amin, -cmin and -mmin switches.

Note:  Had to comment out "symlink" file in t/find2perl.t to run test.

### CODE REVIEW

File: t/find2perl.t [hunk 1]
```diff
@@ -52,11 +52,16 @@ my @test_files =
     (
         { name => "abc" },
         { name => "acc", mtime => time() - HOUR * 48 },
+        { name => "adc", atime => time() - HOUR * 2 },
         { name => "ac", content => "x" x 10 },
         { name => "somedir", type => "d" },
         { name => "link", type => "l", target => "abc" },
-        { name => "symlink", type => "s", target => "brokenlink" },
+
+# Comment out, since trying to create this symlink dies:
```
Comment (Leont) [inline t/find2perl.t]: This should definitely not be part of this commit. Probably it needs to be skipped conditionally based on operating system.

**Comment by Leont** [comment]
Sounds like a reasonable feature request

File: t/find2perl.t (hunk 1 above)
Comment (timgimyee) [reply t/find2perl.t]: Fix in https://github.com/Leont/app-find2perl/pull/4/commits/90f1937bbba5186056f9ebb327237c53273559b2

**Comment by Leont** [comment]
Released as 1.005
